Common/src/main/java/at/petrak/hexcasting/common/casting/actions/spells/OpFlight.kt

```diff
@@ -35,11 +35,20 @@ class OpFlight(val type: Type) : SpellAction {
         val theArg = args.getPositiveDouble(1, argc)
         env.assertEntityInRange(target)
 
-        val cost = when (this.type) {
+        /*val cost = when (this.type) {
             Type.LimitRange -> theArg * MediaConstants.DUST_UNIT
             // A second of flight should cost 1 shard
             Type.LimitTime -> theArg * MediaConstants.SHARD_UNIT
-        }.roundToLong()
+        }.roundToLong()*/
```

Comment:
just remove it, we can get the old code from commit history if necessary

Reply:
yeah I forgot to remove the comment before opening the PR and didn't feel like bothering to make a new commit just for that